planningTime: 120
touchLinks: ["katana_gripper_tool_frame", "katana_gripper_link", "katana_l_finger_link", "katana_r_finger_link", "katana_gripper_tool_agni_frame"] 
eefPosClosed: [-0.495]
eefPosOpen: [0.298]
graspRot: [0, -1,5708, 0]
goalJointTolerance: 0.01 #rad
goalPositionTolerance: 0.02 #m
goalOrientationTolerance: 0.5 #r
graspThroughDistance: 0.02
approachDesiredDistance: 0.10
approachMinDistance: 0.05
liftUpDesiredDistance: 0.02
liftUpMinDistance: 0.005
retreatDesiredDistance: 0.07
retreatMinDistance: 0.03
anglePitchInc: 0.3 # pi/8
anglePitchMax: 0.9 # pi/4
angleYawInc: 0.3 # pi/8
angleYawMax: 0.9 # pi/4
transInc: 0.001
transMax: 0.0
placeXInc: 0.015
placeYZInc: 0.04
placeXMax: 0.08
placeYMax: 0.10
placeZMax: 0.17
heightSkipTop: 0.005
heightSkipBottom: 0.05
heightInc: 0.01